tests/modules/nf-core/deseq2/differential/test.yml

```diff
@@ -1,211 +1,261 @@
+      md5sum: 6332f21889ecac387bb12eeb04f23849
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.normalised_counts.tsv
+      md5sum: 18820fe8057af8f163e9cd8944acd839
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.rlog.tsv
+      md5sum: 1077cf03a8c6a155932eaaed7415b74c
+    - path: output/deseq2/versions.yml
 
 - name: deseq2 differential test_deseq2_differential_spikes
-  command: nextflow run ./tests/modules/nf-core/deseq2/differential -entry test_deseq2_differential_spikes -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/deseq2/differential/nextflow.config
+  command: nextflow run ./tests/modules/nf-core/deseq2/differential -entry test_deseq2_differential_spikes -c ./tests/config/nextflow.config
   tags:
     - deseq2
     - deseq2/differential
   files:
-    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.results.tsv
-      md5sum: 150a32eef482244b47ddc7f7ffb1ad2a
-    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.sizefactors.tsv
-      md5sum: 1708374146b4e8c28fbc78fa292102cd
-    - path: output/deseq2/treatment_mCherry_hND6_sample_number.normalised_counts.tsv
-      md5sum: 1c9c7ee4ca90ff0d91ae16dbfd64fdc9
-    - path: output/deseq2/treatment_mCherry_hND6_sample_number.rlog.tsv
-      md5sum: c28300837df9ca6adeba2a52ea5e181e
-    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.model.txt
-      md5sum: fa05126a58cb67c107d45426b0bdea83
+    - path: output/deseq2/treatment_mCherry_hND6_.deseq2.dispersion.png
+      md5sum: 2c4bd60abd44a03fc63bb88eaa24bf9d
+    - path: output/deseq2/treatment_mCherry_hND6_.deseq2.model.txt
+      md5sum: ecda59670bf0fdcd213b7bf0fc6b7f4c
     - path: output/deseq2/treatment_mCherry_hND6_.deseq2.results.tsv
       md5sum: 88b3230c5a4ad8717c1d0b38c824f1fc
     - path: output/deseq2/treatment_mCherry_hND6_.deseq2.sizefactors.tsv
       md5sum: 1708374146b4e8c28fbc78fa292102cd
     - path: output/deseq2/treatment_mCherry_hND6_.normalised_counts.tsv
-      md5sum: 1c9c7ee4ca90ff0d91ae16dbfd64fdc9
+      md5sum: 6830b58d9d5257a8c4e376064da6e539
     - path: output/deseq2/treatment_mCherry_hND6_.rlog.tsv
       md5sum: c28300837df9ca6adeba2a52ea5e181e
-    - path: output/deseq2/treatment_mCherry_hND6_.deseq2.model.txt
-      md5sum: ecda59670bf0fdcd213b7bf0fc6b7f4c
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.dispersion.png
+      md5sum: b4f873fd54468e0362b75eba41c7433a
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.model.txt
+      md5sum: fa05126a58cb67c107d45426b0bdea83
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.results.tsv
+      md5sum: 150a32eef482244b47ddc7f7ffb1ad2a
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.sizefactors.tsv
+      md5sum: 1708374146b4e8c28fbc78fa292102cd
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.normalised_counts.tsv
+      md5sum: 6830b58d9d5257a8c4e376064da6e539
```

Comment:
why has this md5sum changed?

Reply:
Ah, sorry, because I added the rounding functionality (also mentioned in the PR); I thought it would be practical to have that in this module as well? But I can also turn it off by default.

Reply:
We should always make changes in a backwards-compatible way, so turn any new things off by default.

Reply:
will do